bookkeeper-server/pom.xml

```diff
@@ -84,6 +84,11 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
@eolivelli since it is internal depedency not public, I'd like to first merge this and open another jira to address your concern separately. does that work?